db/migrate/20130827094554_alchemy_two_point_six.rb

```diff
@@ -310,6 +310,88 @@ def up
         t.string "name"
       end
     end
+  end
+
+  def down
```

Comment:
Assignment Branch Condition size for down is too high. [44.72/15]<br>Perceived complexity for down is too high. [21/7]